src/main/java/com/minecolonies/colony/ColonyView.java

```diff
@@ -289,6 +298,21 @@ public IMessage handlePermissionsViewMessage(ByteBuf buf)
     }
 
     /**
+     * Update a ColonyView's workOrders given a network data ColonyView update packet.
+     * This uses a full-replacement - workOrders do not get updated and are instead overwritten.
+     *
+     * @param buf       Network data
+     * @return          null == no response
+     */
+    public IMessage handleColonyViewWorkOrderMessage(ByteBuf buf)
+    {
+        WorkOrderView workOrder = AbstractWorkOrder.createWorkOrderView(buf);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workOrder' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)